Readme.md

```diff
@@ -7,3 +7,5 @@ To run the example use ``mvn exec:java``:
     $ mvn compile exec:java -Dhost=YOUR_CLUSTER_ID.REGION.aws.found.io -Dshield.user="username:password"
 
 Replace `YOUR_CLUSTER_ID` with your cluster id and `REGION` with the region the cluster is started in.
+
+_(Note: by default, if IPv4 and IPv6 are both enabled then it is undetermined which will be used. The system properties `-Dipv4=true|false` and `-Dipv6=true|false` can be used to turn them on/off explicitly. If they are enabled then the network/host should support routing to the Cloud endpoint, eg this is not the case within Docker containers by default)._
```

Comment:
hmm interesting @beiske, now that I re-read the code I see that _should_ be true ... but that definitely _wasn't_ the case yesterday. IPv4 was enabled and working but if IPv6 was present (and wasn't wired up to be routable) then the transport client didn't work